index.html

```diff
@@ -0,0 +1,144 @@
+    <script src="./main.js" defer></script>
+    <script src="./profiles.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <title>Gen Mex Concert 2023 Amazon Originals</title>
+</head>
+<body>
+    <div class="social-media-desktop display-none">
+        <div class="first-container-desktop">
+        <i class="fa-brands fa-facebook-f"></i>
+        <i class="fa-brands fa-twitter"></i>
+        <p>English</p>
+        <p>My Page</p>
+        <p>Logout</p>
+        </div>
+    </div>
+    <div class="nav-bar display-none">
+        <div class="nav-bar-container">
+        <a href="./index.html" class="logo"><img src="images/genmexlogo.jpg" alt="Gen Mex Concert Logo"></a>
+        <nav class="nav-menu">
+        <ul class="nav-options">
+            <li class="nav-link"><a class="link-menu" href="./about.html">About</a></li>
+            <li class="nav-link"><a class="link-menu" href="#">Program</a></li>
+            <li class="nav-link"><a class="link-menu" href="#">Join</a></li>
+            <li class="nav-link"><a class="link-menu" href="#">Sponsor</a></li>
+            <li class="nav-link"><a class="link-menu" href="#">News</a></li>
+            <li class="nav-link"><a id="gmcampaign" class="link-menu" href="#">GM Campaign</a></li>
+        </ul>
+        </nav>
+        </div>
+    </div>
+    <header class="principal-info">
+        <div class="mobile-bar display-none">
+            <nav class="mobile-navmenu">
+            <ul class="mobile-nav-options">
+                <li class="mobile-nav-link"><a class="link-menu" href="./about.html">About</a></li>
+                <li class="mobile-nav-link"><a class="link-menu" href="#">Program</a></li>
+                <li class="mobile-nav-link"><a class="link-menu" href="#">Join</a></li>
+                <li class="mobile-nav-link"><a class="link-menu" href="#">Sponsor</a></li>
+                <li class="mobile-nav-link"><a class="link-menu" href="#">News</a></li>
+                <li class="mobile-nav-link"><a class="link-menu" href="#">GM Campaign</a></li>
+                <li class="mobile-nav-link"><i class="fa-solid fa-xmark"></i></li>
+            </ul>
+            </nav>
+        </div>
+        <div class="main-content">
+        <i id="hamburger" class="fa-solid fa-bars fa-2x"></i>
+        <h3 class="hello-concert">"Hello! Welcome Gen Mex"</h1>
+        <h1 class="concert-title"><b>GEN MEX CONCERT<br>AMAZON ORIGINALS<br> 2023</b></h1>
+        <p class="concert-paragraph"> A joyful event at the NeueHouse Hollywood.
+        The event brings together a new generation of mexican fresh stars who will delight 
+        the audience with some of their best hits.
+        </p>
+        <p class="date-and-time">2023.03.03(FRI) ~ 05(SUN)</p>
+        <p class="place"><b>@NeueHouse Hollywood, Columbia Square, USA</b></p>
+        </div>
+    </header>
```

Comment:
- [ ] Have you thought about how nicely it will improve your aesthetics if you include a great `background-image`? Kindly add one please, I think your users will like it.

Reply:
done